httpserver/http_connection.cpp

```diff
@@ -1104,6 +1117,7 @@ void HttpClientSession::start() {
 
 void HttpClientSession::trigger_callback(SNodeError error,
                                          std::shared_ptr<std::string>&& body) {
+    LOKI_LOG(trace, "trigger callback");
```

Comment:
Not important but would like to try have all new logs (and retroactively if we see them) start with a capital letter... 
# :confetti_ball: